src/midi/midi_rtmidi.cpp

```diff
@@ -37,18 +36,22 @@ void MidiInRt::close()
         m_midiin->closePort();
 }
 
-void MidiInRt::open(unsigned port)
+bool MidiInRt::open(unsigned port)
 {
     RtMidiIn *midiin = lazyInstance();
-    close();
+    m_midiin->closePort();
+    m_errorSignaled = false;
     midiin->openPort(port, "MIDI input");
+    return !m_errorSignaled;
 }
 
-void MidiInRt::openVirtual()
+bool MidiInRt::openVirtual()
 {
     RtMidiIn *midiin = lazyInstance();
-    close();
+    m_midiin->closePort();
+    m_errorSignaled = false;
     midiin->openVirtualPort("MIDI input");
```

Comment:
Why not to call this port as "OPL3 Bank Editor MIDI Input"? As this "MIDI Input" pseudo-device was shown at me

Reply:
It's a redundant display in jack midi but overall you're right about it. I'll change it.